test/framework/common.go

```diff
@@ -103,7 +103,15 @@ func WebServerApplicationImageUpdateTest(clt client.Client, ctx context.Context,
 		time.Sleep(time.Second * 5)
 	}()
 
-	return webServerApplicationImageUpdateTest(clt, ctx, t, webServer, newImageName, testURI)
+	webServerApplicationImageUpdateTest(clt, ctx, t, webServer, newImageName, testURI)
```

Comment:
you need to check err here... And return if error.

Reply:
right.. done.